main.go

```diff
@@ -22,12 +22,39 @@ type SlackMessage struct {
 	Channel   string `json:"channel"`
 }
 
+func CreateSlackMessage(eventData []byte) SlackMessage {
+	// Assumes data is in the form irc/name/icon␀msg
+	i := bytes.IndexByte(eventData, byte('\x00'))
+	if i == -1 {
+		return SlackMessage{
```

Comment:
Could you add a comment explaining what is happening in this case? "The message lacks a nul byte, so just send the message alone".